### ISSUE TITLE
Faded launch.json properties are not readable on a light theme

### ISSUE THREAD

**Issue by Tyriar** [Microsoft/vscode#14868]
- VSCode Version: Code 1.7.0 (9e7d70052908b633e0d4f71e70b26714bd1d2f10, 2016-11-02T14:27:44.073Z)
- OS Version: Linux x64 4.8.0-26-generic
- Extensions:

|Extension|Author|Version|
|---|---|---|
|EditorConfig|EditorConfig|0.3.4|
|ActiveFileInStatusBar|RoscoP|1.0.2|
|lorem-ipsum|Tyriar|1.0.0|
|sort-lines|Tyriar|1.1.0|
|theme-glacier|Tyriar|0.0.10|
|vscode-terminal-here|Tyriar|0.0.5|
|vscode-svgviewer|cssho|1.2.0|
|vscode-eslint|dbaeumer|1.0.8|
|git-project-manager|felipecaputo|0.1.15|
|annotator|ryu1kn|0.10.1|
|vscode-header-source|ryzngard|1.0.4|
|gitblame|waderyan|1.3.0|

---

The contrast is so low that many users will have difficulty reading.

![image](https://cloud.githubusercontent.com/assets/2193314/19941441/58a293aa-a0ed-11e6-87c0-b9c191cfd6de.png)

Having pre-defined colors like this is probably not the best idea. using opacity might be more effective to ensure it looks at home in other themes.

**Comment by isidorn** [comment]
Using opacity might be better however that is currently not supported by our vscode decoration api (since this is done from an extension).
The screenshot you attached is not a default light theme, in my light theme it looks very readable. Though I agree we could play more with colors to make it more accessible. 
Assigning to @bgashler1 for that, here is a [code pointer](https://github.com/Microsoft/vscode/blob/master/extensions/configuration-editing/src/extension.ts#L13) where a special color could be added for light and dark theme

**Comment by akamud** [comment]
Why are those properties faded? Any special reason? Couldn't find anything in the JSON editing documentation.

**Comment by isidorn** [comment]
@akamud because those properties should not be changed by the user. The user shuold focus on the non faded properties. At least that is the reasoning

**Comment by Tyriar** [comment]
Contrast isn't good enough for all the light themes we ship, we should do something about this

**Comment by isidorn** [comment]
I have changed the color to a stronger gray `9e9e9e`
I have tried all the default themes and it looks good enough imho
In case you do not agree feel free to pick a different shade of gray. Code pointer https://github.com/Microsoft/vscode/blob/master/extensions/configuration-editing/src/extension.ts#L17
